src/mod2c_core/modl.c

```diff
@@ -87,8 +87,10 @@ static void openfiles(char* given_filename, char* output_dir) {
   modprefix = strdup (given_filename);                      // we want to keep original string to open input file
   char* first_ext_char = strrchr(modprefix, '.');           // find last '.' that delimit file name from extension
 
+  Sprintf(input_filename, "%s", given_filename);
```

Comment:
given_filenae was not null terminated?

Reply:
?

Reply:
hum, no, we use input_filename line 116 without being initialized if the file exists

Reply:
Oh I see!